src/lib/components/UploadBtn.svelte

```diff
@@ -3,21 +3,26 @@
 
 	export let classNames = "";
 	export let files: File[];
-	let filelist: FileList;
 
-	$: if (filelist) {
-		files = Array.from(filelist);
-	}
+	/**
+	 * Due to a bug with Svelte, we cannot use bind:files with multiple
+	 * So we use this workaround
+	 **/
+	const onFileChange = (e: Event) => {
+		if (!e.target) return;
+		const target = e.target as HTMLInputElement;
+		files = [...files, ...(target.files ?? [])];
+	};
 </script>
 
 <button
-	class="btn relative h-8 rounded-lg border bg-white px-3 py-1 text-sm text-gray-500 shadow-sm transition-all hover:bg-gray-100 dark:border-gray-600 dark:bg-gray-700 dark:text-gray-300 dark:hover:bg-gray-600 {classNames}"
+	class="btn relative h-8 rounded-lg border bg-white px-3 py-1 text-sm text-gray-500 shadow-sm hover:bg-gray-100 dark:border-gray-600 dark:bg-gray-700 dark:text-gray-300 dark:hover:bg-gray-600 {classNames}"
 >
 	<input
-		bind:files={filelist}
 		class="absolute w-full cursor-pointer opacity-0"
 		type="file"
-		accept="image/*"
+		on:change={onFileChange}
+		accept="*/*"
```

Comment:
should we make it bit more restricted like onlt image & documents (pdf, text, ...) ?